extensions/wikia/ARecoveryEngine/js/sourcePoint.js

```diff
@@ -51,12 +60,17 @@ define('ext.wikia.aRecoveryEngine.recovery.sourcePointHelper', [
 	}
 
 	function isBlocking() {
-		log(['isBlocking', !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking)], 'debug', logGroup);
-		return isSourcePointRecoveryEnabled() && !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking);
+		var isBlocking = !!(win.ads && win.ads.runtime.sp && win.ads.runtime.sp.blocking);
+
+		log(['isBlocking', isBlocking], 'debug', logGroup);
+		return isBlocking;
 	}
 
-	function isSourcePointRecoverable(slotName, recoverableSlots) {
-		return isSourcePointRecoveryEnabled() && recoverableSlots.indexOf(slotName) !== -1;
+	function isSlotRecoverable(slotName) {
+		var result = isSourcePointRecoveryEnabled() && recoverableSlots.indexOf(slotName) !== -1;
```

Comment:
What do you think of renaming `isSourcePointRecoveryEnabled()` to just `isEnabled()`? :-p